ppdet/modeling/backbones/bifpn.py

```diff
@@ -41,7 +41,8 @@ def __call__(self, inputs, name=''):
             groups=self.num_chan,
             param_attr=ParamAttr(
                 initializer=Xavier(), name=name + '_dw_w'),
-            bias_attr=False)
+            bias_attr=False,
+            use_cudnn=False)
```

Comment:
according to previous benchmark, on cudnn 7.6+ depthwise is indeed faster, especially during inference